package/cluster/eks/definition.yaml

```diff
@@ -27,6 +27,37 @@ spec:
                 type: object
                 description: EKS configuration parameters.
                 properties:
+                  region:
+                    type: string
+                    default: us-west-2
+                    description: AWS region
```

Comment:
Same as above. Please mention the cloud provider documentation URL in the description to make it clear where the region list is coming from.